charts/rancher-wins-upgrader/templates/_helpers.tpl

```diff
@@ -30,7 +30,7 @@ provider: kubernetes
 {{- end -}}
 
 {{- define "winsUpgrader.winsHostPath" -}}
-{{ .Values.prefixPath | replace "\\\\" "/" }}etc/rancher/wins
+{{ .Values.global.cattle.rkeWindowsPrefixPath | replace "\\\\" "/" }}etc/rancher/wins
```

Comment:
Do you think we need this replace statement anymore? Or should we just drop this and add a comment on rkeWindowsPathPrefix that it needs to use forward slashes?

Reply:
You probably know the best option for that. I am happy to make whatever changes you think are best.